.github/actions/gcloud/action.yml

```diff
@@ -0,0 +1,47 @@
+name: gcloud
+description: Configure gcloud
+inputs:
+  cloud_sdk_service_account_key:
+    required: true
+  gcp_project:
+    required: true
+  gcp_zone:
+    required: true
+  create:
+    required: false
+    description: Set to true to create a new cluster which will be destroy after the job finishes. Leave empty to just configure the gcloud tooling.
+  name:
+    required: false
+    description: Cluster name; required only if 'create' is true.
+  machine_type:
+    required: false
+    default: n1-standard-2
+  num_nodes:
+    required: false
+    default: 1
+  release_channel:
+    required: false
+    default: rapid
+  cluster_version:
+    required: false
+    default: ''
+  preemptible:
+    required: false
+    default: true
+  enable_network_policy:
+    required: false
+    default: true
+  enable_stackdriver:
+    required: false
+    default: false
+  enable_basic_auth:
+    required: false
+    default: false
+  enable_legacy_auth:
+    required: false
+    default: false
+runs:
+  using: node12
+  main: dist/index.js
+  post: dist/index.js
+  post-if: always()
```

Comment:
Does this `post_if` condition ensure the action runs again and hits the `isPost` branch that deletes the cluster?

Reply:
yep, that's correct. I did test having the cluster creation job fail, and the deletion post step does get triggered.